bin/flush_cache

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+require_relative '../deployment'
+require 'cdo/aws/cloudfront'
+
+require 'sshkit'
+require 'sshkit/dsl'
+include SSHKit::DSL
+
+CDO.log.info 'Flushing Varnish caches on frontends...'
+on CDO.app_servers.values do |host|
+  execute "sudo varnishadm -T localhost:6082 -S /etc/varnish/secret 'ban obj.status ~ .'"
+end
+CDO.log.info 'Flushing CloudFront caches (up to 15 min)...'
+AWS::CloudFront.invalidate_caches
```

Comment:
there shouldn't be any race condition- the varnish cache ban takes effect as soon as it's executed (the ssh call does block, but the call returns 'instantly').